apps/dashboard/app/views/batch_connect/session_contexts/_path_selector.html.erb

```diff
@@ -6,23 +6,25 @@ end
 
 <%= label attrib.id, "#{field_options[:label]}" %>
 <%= text_field :batch_connect_session_context, attrib.id, class: 'form-control' %>
-<button type="button" class="btn btn-primary mt-2" data-toggle="modal" data-target="#modalPathSelector" id='select-nav-path'>
+<button type="button" class="btn btn-primary mt-2" data-toggle="modal" data-target="#modal-path-selector" id='select-nav-path'>
   Select Path
 </button>
 <span id='path_selector_home_dir' class="d-none"><%= Dir.home %></span>
 <span id='path_selector_url' class="d-none"><%= files_path %></span>
+<span id='path_selector_options' class="d-none"><%= attrib.opts[:options].is_a?(Array) ? attrib.opts[:options].join(",") : '' %></span>
 
 <!-- Modal -->
-<div class="modal fade h-75" id="modalPathSelector" tabindex="-1" role="dialog" aria-labelledby="modalPathSelector" aria-hidden="true">
+<div class="modal fade h-75" id="modal-path-selector" tabindex="-1" role="dialog" aria-labelledby="modal-path-selector" aria-hidden="true">
```

Comment:
As long as we're updating the ids, can we use underscores as the separator?  We have that listed as our javascript style.

https://github.com/OSC/ondemand/blob/master/CONTRIBUTING.md#javascript-style

Reply:
In fact we may even need a more unique id given that there could be 2 or more of these widgets in a form.

Reply:
Not sure if you reviewed this sooner, than later.  But, this change was already in place.